prusti-tests/tests/cargo_verify/veribetrfs/src/base/option.rs

```diff
@@ -0,0 +1,36 @@
+use prusti_contracts::*;
+
+pub enum Opt<T> {
```

Comment:
You can use the built-in Rust `Option` type by giving it specs as shown [here](https://github.com/viperproject/prusti-dev/blob/e51e0c21b5f0ec5ad7e7ca9b3b8eda322965c0cb/prusti-tests/tests/verify/pass/extern-spec/option.rs).